hr_timesheet_sheet/i18n/nl_BE.po

```diff
@@ -731,7 +731,7 @@ msgstr "Week"
 #: code:addons/hr_timesheet_sheet/models/hr_timesheet_sheet.py:354
 #, fuzzy, python-format
 msgid "Week %s"
-msgstr "Week"
+msgstr "Week %s"
```

Comment:
This translation has become useless

Reply:
And it will be purged by some bot?

Reply:
Yes, but losing current translations for word `Week`, so IMO it's better to strip `%s` from all existing translations.

Reply:
What about this?

Reply:
I've removed bad translation and made it as-is in other translations. Hopefully useless string will be removed by bot